src/status_im/contexts/shell/share/view.cljs

```diff
@@ -52,7 +52,7 @@
         abbreviated-url   (address/get-abbreviated-profile-url
                            universal-profile-url)
         emoji-hash-string (string/join emoji-hash)]
-    [:<>
+    [rn/scroll-view
```

Comment:
perhaps we may ask Design team this question,

"What should be done for small devices in this case?"
we could also make the whole parent view scrollable instead of just the inner tab view.

cc @status-im/design-team

Reply:
> "What should be done for small devices in this case?" we could also make the whole parent view scrollable instead of just the inner tab view.
> 
Scrolling should only affect the inner tabs :)

Reply:
Hi @Francesca-G : thank you, Just to confirm, is this applicable also for vertical scroll? 
I know we keep scrolling inside tab view for wallet accounts and that's horizontal scrolling.

Reply:
> Hi @Francesca-G : thank you, Just to confirm, is this applicable also for vertical scroll? I know we keep scrolling inside tab view for wallet accounts and that's horizontal scrolling.

Yes, I was talking about vertical scrolling 👍

Reply:
Thanks for confirming.